### DISCUSSION TITLE
Latest release

### DISCUSSION THREAD

**Discussion by karimdabbagh** [TomasBouda/PoShLog#38] [General]
When are you planning to release the latest changes? Foe example, JsonFormatter for console does not exist in the latest release yet.

Thanks,
Karim

**Comment by TomasBouda** [comment]
Hi @karimdabbagh,
I know it's been a while since your question... I have very little spare time, but Im still planing ressurecting this project, I just don't want to promise anything right now😃 I would love to convert to binary module, add few functions and release v3.0.0
